TreeDB/internal/mappedresource/manager_test.go

```
@@ -52,6 +57,47 @@ func TestAcquireBytesReleaseUpdatesAccountingAndPins(t *testing.T) {
 
+func TestMappedResourceStatsSnapshotIsDeepCopy(t *testing.T) {
+	mgr := NewManager()
+	h, err := mgr.AcquireBytes(testKey(), testScope(), SourceHeapCopy, []byte("0123456789abcdef"), AcquireOptions{
+		Reason:         "snapshot",
+		ValidationMode: ValidationCachedVerify,
+		FallbackReason: FallbackReadAt,
+	})
+	if err != nil {
+		t.Fatalf("AcquireBytes: %v", err)
+	}
+	defer h.Release()
```

golangci-lint (2.12.2)

[error] Error return value of `h.Release` is not checked

(errcheck)